### ISSUE TITLE
(Option to) Hide player's video box

### ISSUE THREAD

**Issue by benbarbour** [saif-ellafi/foundryvtt-minimal-ui#20]
I have only a few players that send me video, and I like to make them fairly big. Without popping out the video frames (which has it's own issues when the windows resizes) this results in a lot of wasted space. It would be great to have the option to hide a player's video box, on a player-by-player basis.

**Comment by benbarbour** [comment]
Thanks! I realized after filing this that although the suggestion fits with the theme of this module, maybe this exact issue is a better suggestion for a core feature. Maybe I'll throw a suggestion at the core team.

**Comment by saif-ellafi** [comment]
Yes, at a first time when I did the camera hiding feature for players, was already a "grey" area with something that Foundry should provide!

It is a great idea to provide the suggestion to core.

**Comment by saif-ellafi** [comment]
Will wait for a little longer to gather whether other users are also interested in this. It is quite the effort to do it. If nothing happens, will close this item eventually, as it is not yet on the short-term plans.

**Comment by saif-ellafi** [comment]
Took a year but I finally did it - @benbarbour this feature is in 1.4.18
